packages/blockchain/xrp/src/lib/xrp.sdk.spec.ts

```diff
@@ -0,0 +1,19 @@
+import {TatumXrpSDK} from './xrp.sdk'
+import {REPLACE_ME_WITH_TATUM_API_KEY, TEST_DATA} from "@tatumio/shared-testing";
+
+describe('TatumXrpSDK', () => {
+  const sdk = TatumXrpSDK({apiKey: REPLACE_ME_WITH_TATUM_API_KEY})
+
+  describe('Wallet', () => {
+    it('Generate wallet', async () => {
+      try {
+        const {address, secret} = await sdk.blockchain.wallet()
+        expect(address).toMatch(TEST_DATA.XRP.ADDRESS_REGEX)
+        expect(secret).toMatch(TEST_DATA.XRP.SECRET_REGEX)
+      } catch (e) {
+        console.log("Generate XRP wallet failed: ", e)
+        expect(e).not.toBeDefined()
```

Comment:
if the exception will arise - test will fail automatically

Reply:
Also please check `walletTestFactory`

I suppose we will have the same tests for XRP and XLM, right? 
So maybe it makes sense to create `xrpLikeWalletTestFactory` and add more cases?

Reply:
You are right, test will fail, but the error message was not sufficient, I needed to log the error to get more information contained in logged error, in this cas it was the `body` content.

About the factory function, I can see your point, I will do that.

Reply:
What other cases do think are important to include?